application/locale/ru_RU/LC_MESSAGES/messages.po

```diff
@@ -234,27 +235,27 @@ msgstr ""
 #: application/controllers/Timeline.php:101
 #: application/controllers/Timeline.php:104
 msgid "Log View"
-msgstr ""
+msgstr ":ehyfk"
 
 #: application/controllers/Awards.php:457
 msgid " and sat "
-msgstr ""
+msgstr " и спутник"
 
 #: application/controllers/Awards.php:460
 msgid " and orbit type "
-msgstr ""
+msgstr " и тип орбиты"
 
 #: application/controllers/Awards.php:464
 msgid " and propagation "
-msgstr ""
+msgstr " и тип прохождения"
 
 #: application/controllers/Awards.php:467
 msgid " and mode "
-msgstr ""
+msgstr " и режим модуляции"
 
 #: application/controllers/Awards.php:470
 msgid " and "
-msgstr ""
+msgstr " и"
```

Comment:
here is missing a space 
```sugestion
msgstr " и "
```

Reply:
Ok. Thanks.
I'll check in two hours. It's very difficult to do this from a cell phone.